AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -271,11 +271,18 @@ - (void)checkLatestRelease:(NSString *)updateToken
 
     // Check if sender is still waiting for a response of the previous request.
```

Comment:
I don't think we need this comment anymore. What do you think?

Reply:
Removed.